Merge/merge.c

```diff
@@ -0,0 +1,51 @@
+void merge(int List[], int Base, int index, int Limit);
+
+void merge_sort(int List[], int Base, int Limit) {
+
+ int index;
+
+ if(Base < Limit) {
+    index = (Base + Limit) / 2;
+    merge_sort(List, Base, index);
+    merge_sort(List, index + 1, Limit);
+    merge(List, Base, index, Limit);
+ }   
+}
+
+void merge(int List[], int Base, int index, int Limit) {
+    int array[100];
```

Comment:
create global constant MAX_ARRAY_SIZE with value 100 and set array limit as MAX_ARRAY_SIZE